stacks/triliovault-operator/upgrade.sh

```diff
@@ -23,13 +28,13 @@ echo "Upgrading Triliovault operator..."
 
 if [ -z "${MP_KUBERNETES}" ]; then
   # use local version of values.yml
-  ROOT_DIR=$(git rev-parse --show-toplevel)
-  values="$ROOT_DIR/stacks/triliovault-operator/values.yml"
-  TVM="$ROOT_DIR/stacks/$STACK/triliovault-manager.yaml"
+  values="$TVK_PATH/values.yml"
+  TVM="$TVK_PATH/triliovault-manager.yaml"
 else
   # use github hosted master version of values.yml
   values="https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/triliovault-operator/values.yml"
   TVM="https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/triliovault-operator/triliovault-manager.yaml"
+  TVK_LICENSE_FILE="$TVK_PATH/tvk_install_license.yaml"
```

Comment:
same

Reply:
@bhagirathhapse Please note Dhruv's comment above. All paths must be fully qualified urls.

Reply:
@keladhruv @bikram20 I misunderstood the change that Dhruv was suggesting. I have made those changes now and code is pushed. Please review and let me know.
Thanks!